src/helpers/nft.ts

```diff
@@ -54,17 +125,17 @@ export const secretNftEncryptAndUploadFile = async <TNFT, TMedia>(
  * @param secretNftMetadata  Secret NFT metadata (Title, Description).
  * @param ipfsClient         A TernoaIPFS instance.
  * @param ownerPair          Account of the secret NFT's owner.
- * @param clusterId          The TEE Cluster id.
+ * @param clusterId          The TEE Cluster id. Default is set to cluster id 0.
  * @param royalty            Percentage of all second sales that the secret NFT creator will receive. Default is 0%. It's a decimal number in range [0, 100].
  * @param collectionId       The collection to which the secret NFT belongs. Optional Parameter: Default is undefined.
  * @param isSoulbound        If true, makes the secret NFT intransferable. Default is false.
  * @returns                  A JSON including both secretNftEvent & TEE enclave response (shards datas and description).
  */
-export const mintSecretNFT = async <T>(
+export const mintSecretNFT = async (
```

Comment:
We should also add a `waitUntil: WaitUntil` parameter to this function.

Reply:
We can also specify the on-chain minting on the typescript header function description:
`Encrypts your data to create a secret NFT on-chain and uploads your key's shards on a TEE.`